procurement/templates/procurement/contract_detail.html

```diff
@@ -1,77 +1,134 @@
-            </div>
+      {% if tender.published %}
+        <p>Published: {{ tender.published }}</p>
+      {% endif %}
 
-            <h5>Classifications</h5>
-            <p>
-            {% for classification in tender.tenderclassification.all %}
-                {{ classification.classification.description }}{% if not forloop.last %} &middot; {% endif %}
-            {% endfor %}
-            </p>
-
-            <div class="row gx-md-5 mt-3 mt-md-4">
-                <div class="col-sm-auto">
-                    <small class="d-block text-muted fw-bold">Total cost</small>
-                    <span class="h4">£{{ tender.value|floatformat:"0"|intcomma }}</span>
-                </div>
-                <div class="col-sm-auto">
-                    <small class="d-block text-muted fw-bold">Estimated emissions</small>
-                    <span class="h4">??</span>
+        <h2 class="h5 mt-4 mt-md-5 mb-3">Time until this contract ends</h2>
+
+      {% for award in tender.awards.all %}
+        <div class="progress mt-3 mt-md-4">
+            <div class="progress-bar bg-danger" style="width: {{ award.contract_percent_complete }}%"></div>
+        </div>
+        <div class="d-flex flex-wrap">
+          {% if award.contract_ended %}
+            <strong class="mt-2 me-auto">Ended on {{ award.end_date }}</strong>
+          {% elif not award.contract_started %}
+            <strong class="mt-2 me-auto">Starts on {{ award.start_date }}</strong>
+          {% else %}
+            <strong class="mt-2 me-auto">{{ award.contract_time_remaining_desc }}</strong>
+          {% endif %}
+            <span class="mt-2">{{ award.contract_length_desc }}</span>
+        </div>
+      {% empty %}
+        <div class="d-flex flex-wrap">
+          state: {{ tender.state }}
+          <p>
+          {% if tender.state == 'active' %}
+              This contract has yet to be awarded. The tender process ends on {{ tender.end_date }}.
+          {% elif tender.state == 'cancelled' %}
+              This contract was cancelled.
+          {% elif tender.state == 'unsuccesful' %}
+              The contracting process for this tender was unsuccesful.
+          {% endif %}
+          </p>
+        </div>
+      {% endfor %}
+
+        <h2 class="h5 mt-4 mt-md-5 mb-3">Total cost</h2>
+
+        <p>£{{ tender.value|floatformat:"0"|intcomma }}</p>
+
+        <!-- <h2 class="h5 mt-4 mt-md-5 mb-3">Estimated emissions</h2> -->
+        <!-- <h2 class="h5 mt-4 mt-md-5 mb-3">Social value</h2> -->
+
+        <h2 class="h5 mt-4 mt-md-5 mb-3">More like this</h2>
+        <ul class="list-inline">
+          {% for classification in tender.tenderclassification.all %}
+          {% if classification.classification.group  %}
+                <li class="list-inline-item">
+                    <a href="{% url 'home' %}{% urlparams classification=classification.classification.group  %}">
+                        {{ classification.classification.description }} contracts
```

Comment:
Does this want to be `{{ classification.classification.group }}`, so that it's based on groups (which is what is being filtered on) as opposed to the more granular classification descriptions?

Reply:
I hadn’t really grokked the difference between the classification groups and the classifications themselves, until our call earlier. Agree we almost _always_ want to be exposing the _groups_ to users, and not the underlying classifications themselves. From what we heard in the prototyping, people usually think in terms of these top level "types" of contract, not the specific codes.